src/dataLayer/mongo/user.js

```diff
@@ -66,3 +66,24 @@ export async function createUser(root, vars, ctx) {
     return exists[0];
   }
 }
+
+export async function deleteUser(root, vars, ctx) {
+  const { decoded } = verifyWebToken(ctx);
+  const { accountLinkId } = vars;
+  const loggedInId = decoded[namespace + 'accountLinkId'];
+
+  const exists = await asyncErrorHandler(
+    doesExist(UserModel, { accountLinkId })
+  );
+  if (isEmpty(exists)) {
+    return {
+      statusCode: 404,
+      message: 'There is no account with this accountLinkId'
+    };
+  }
+  await asyncErrorHandler(UserModel.deleteOne({ accountLinkId }));
+  return {
```

Comment:
Don't think this has to return anything. If anything it should be the ID of the deleted user.

Reply:
I'm not sure, if anything we should return the deleted user. The caller should make sure their call was successful by asserting there were no errors.

Reply:
OK